docker/test_generate_makefile.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate Makefile test."""
+
+from unittest.mock import patch, call
+
+from docker import generate_makefile
+
+
+@patch('builtins.print')
+def test_print_makefile_build(mocked_print):
+    """Tests result of a makefile generation for an image."""
+
+    name = 'afl-zlib-builder-intermediate'
+    image = {
+        'tag': 'builders/afl/zlib-intermediate',
+        'context': 'fuzzers/afl',
```

Comment:
Can you rename `context` to `path` everywhere you use it in this manner. That is how the docker help string describes it. `context` can mean anything.

Reply:
Done